docs/setup/env-setup.md

```diff
@@ -87,6 +87,24 @@ Without setting the enviromnent minikube might not find the docker images when d
 1. run the image build from the yunikorn-k8shim repository root: `make image`
 1. deploy the scheduler as per the normal instructions.
 
+## Debug code locally
+
+Note, this instruction requires you have GoLand IDE for development.
+
+In GoLand, go to yunikorn-k8shim project. Then click "Run" -> "Debug..." -> "Edit Configuration..." to get the pop-up configuration window,
+
+![Debug Configuration](../images/goland_debug.jpg)
+
+The highlighted fields are the configurations you need to add. These include:
+
+- Run Kind: package
+- Package path: point to the path of `pkg/shim` package
+- Working directory: point to the path of the `conf` directory, this is where the program loads configuration file from
+- Program arguments: specify the arguments to run the program, such as `-kubeConfig=/path/to/.kube/config -interval=1s -clusterId=mycluster -clusterVersion=0.1 -name=yunikorn -policyGroup=queues -logEncoding=console -logLevel=-1`.
```

Comment:
This text hides the `/path/to/.kube` reference and I did not see it until I copied it in and walked over each parameter to see what it did.